packages/zoltan2/core/src/algorithms/color/Zoltan2_AlgHybridD1.hpp

```diff
@@ -773,7 +773,8 @@ class AlgDistance1 : public Algorithm<Adapter>
         deep_copy(verts_to_send_size, verts_to_send_size_host);
         //set the old ghost colors
 	//get the colors from the femv
-        Kokkos::View<int**, Kokkos::LayoutLeft> femvColors = femv->template getLocalView<memory_space>(Tpetra::Access::ReadWrite); // Partial write
+        Kokkos::View<int**, Kokkos::LayoutLeft, device_type> femvColors =
+	  femv->template getLocalView<memory_space>(Tpetra::Access::ReadWrite); // Partial write
```

Comment:
All other calls to getLocalView were changed to take a device_type instead of a memory_space.  Should this call change as well?